### DISCUSSION TITLE
I have a doubt about heroku deploy

### DISCUSSION THREAD

**Discussion by thesanjithkumar** [DenverCoder1/github-readme-streak-stats#284] [Q&A] [answered]
If I deploy On my own heroku server and if someone else will be able to use my heroku link?

**Comment by DenverCoder1** [comment, marked as answer]
If the code you are hosting is identical then anyone will be able to use your link.

I don't think it's a big deal (I have mine use by tens of thousands :smile:)

I suppose if you want to prevent that, you could fork the repo and hardcode your own username in place of `$_REQUEST["user"]` so it only works for your profile.
